src/models/thelio-b4/external-overview.md

```diff
@@ -0,0 +1,18 @@
+# Thelio (External Overview)
+
+## Back overview:
+
+![Back Ports](./img/ports-back.webp)
```

Comment:
I believe the ethernet port should be labeled `2.5Gb/s` or `2.5Gb` rather than `2.5GiB`. GiB is gibibyte, and gibibyte != gigabyte != gigabit.

Reply:
You're right, I used the wrong abbreviation. [Microsoft's styling guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms) and the motherboard manual use `Gbps`, so I'll go with that. (Intel ARK uses `GbE`.)

Reply:
Fixed with https://github.com/system76/tech-docs/pull/153/commits/21c55ff87f2b786a5bf7bbd8cdda2f6d69b43dd0.